c2corg_ui/static/js/addassociation.js

```diff
@@ -96,6 +96,7 @@ app.AddAssociationController.prototype.associate = function(doc) {
   }
 
   this.api_.associateDocument(parentId, childId).then(function() {
+    doc['new'] = true;
```

Comment:
Do you need this? Isn't it already added in ``app.Document.prototype.pushToAssociations``?